core/graph/src/main/java/uk/gov/gchq/gaffer/graph/Graph.java

```diff
@@ -246,11 +246,21 @@ private void updateOperationChainView(final Operations<?> operations) {
                 View opView = ((OperationView) operation).getView();
                 if (null == opView) {
                     opView = config.getView();
-                } else if (!(opView instanceof NamedView) && !opView.hasGroups()) {
+                } else if (!(opView instanceof NamedView) && !opView.hasGroups() && !opView.isAllEdges() && !opView.isAllEntities()) {
                     opView = new View.Builder()
                             .merge(config.getView())
                             .merge(opView)
                             .build();
+                } else if (opView.isAllEdges() || opView.isAllEdges()) {
```

Comment:
This should be:

```java
else if (opView.isAllEdges() || opView.isAllEntities())
```

Please can you add another test to check it works for both edges and entities.